edge-hub/watchdog/src/main.rs

```diff
@@ -36,7 +36,7 @@ fn main() -> Result<()> {
     let edgehub_handle = run(
         "Edge Hub",
         "dotnet",
-        vec!["/app/Microsoft.Azure.Devices.Edge.Hub.Service/Microsoft.Azure.Devices.Edge.Hub.Service.dll".to_string()],
```

Comment:
This was needed before because the build pipeline was injecting a faulty build argument, which had the side effect of copying the exe containing folder itself ```Microsoft.Azure.Devices.Edge.Hub.Service```. We no longer need this.